mm-go-irckit/userbridge.go

```diff
@@ -137,11 +137,13 @@ func (u *User) handleDirectMessageEvent(event *bridge.DirectMessageEvent) {
 		prefix := u.prefixContext(prefixUser, event.MessageID, event.ParentID, event.Event)
 
 		switch {
-		case u.v.GetBool(u.br.Protocol() + ".prefixcontext"):
+		case u.v.GetBool(u.br.Protocol()+".prefixcontext") && strings.HasPrefix(event.Text, "\x01"):
```

Comment:
this isn't something detected by the linter, right?

Reply:
This is a bug after the nestif fixes in handleChannelMessages. So you get message/thread/prefix twice. e.g.

    |07:46 * hloeung Testing [jabpxsdo7pge7gn7yidgm8ki6w] A [@@jabpxsdo7pge7gn7yidgm8ki6w]

Do you prefer that as a separate commit?

Reply:
No it's okay, was just wondering.